src/Avalonia.Controls/ContextMenu.cs

```diff
@@ -57,8 +57,18 @@ public class ContextMenu : MenuBase, ISetterValue, IPopupHostProvider
         /// <summary>
         /// Defines the <see cref="PlacementMode"/> property.
         /// </summary>
-        public static readonly StyledProperty<PlacementMode> PlacementModeProperty =
-            Popup.PlacementModeProperty.AddOwner<ContextMenu>();
+        [Obsolete("Use the Placement property instead.")]
+        public static readonly StyledProperty<PlacementMode> PlacementModeProperty = PlacementProperty!;
```

Comment:
Isn't it actually null here? `PlacementProperty` is defined below, so I think it's initialized after this one.

Reply:
Lol, It was late and I didn't think...